### ISSUE TITLE
Advanced script : Model.AddTable() bug ?

### ISSUE THREAD

**Issue by didierterrien** [otykier/TabularEditor#464]
Hello Daniel,
Am I wrong or this simple advanced script doesn't work ?
`Model.AddTable("newtable");`
Thanks in advance for your reply

**Comment by otykier** [comment]
It works on my side.
- Which version of Tabular Editor are you using?
- Are you running this script on an empty model or a model loaded from Power BI / Analysis Services?
- Are you getting an error message or does the table simply not show up?

**Comment by didierterrien** [comment]
My version is the latest (2.9.8)
I run it on a bim file (file is attached)
There is no message and the table doesn't show up

Strange behavior :
1. It didn't work
2. I moved the line in my code then it worked
3. Then I moved it back and it worked also 

`(2)
var Visuals_labels_file = @"C:\Users\didier.terrien\Desktop\PBI translations\TE visuals labels.tsv";

var Target_table_name = "Visuals translations measures";

(1) & (3)`

Now I cannot reproduce the error. Sorry for the disturbance

[PBI translations (2).zip](https://github.com/otykier/TabularEditor/files/4581394/PBI.translations.2.zip)
